src/Psalm/Internal/PluginManager/PluginList.php

```diff
@@ -34,9 +35,11 @@ public function getEnabled(): array
     {
         if (!$this->enabled_plugins) {
```

Comment:
This is likely an error: it's claiming to be memoized here, but if no plugins are enabled, it will retry since an empty array is also falsy.

Reply:
Should probably be `if (null === $this->enabled_plugins)`

Reply:
Yes, but it's unrelated to this PR.